letsencrypt-auto-source/tests/centos6_tests.sh

```diff
@@ -74,6 +89,9 @@ echo "PASSED: Successfully upgraded to Python3.4 using letsencrypt-auto < 0.37.0
 # now bootstrap from current letsencrypt-auto, that will install python3.6 from SCL
 "$LE_AUTO" --no-self-upgrade -n > /dev/null 2> /dev/null
 
+# enable SCL rh-python36
+. scl_source enable rh-python36
```

Comment:
I do this on purpose because the end of this script is:
```
# test using python3
pytest -v -s certbot/letsencrypt-auto-source/tests
```

Before the python 3.4 from EPEL installed by the previous version of `certbot-auto` was used. Now it is the duty of python 3.6 from SCL, but to be accessible, `rh-python36` needs to be enabled.

Reply:
I believe the `# test using python3` is wrong and the tests are still being run using Python 2.6. See https://github.com/certbot/certbot/issues/5632. This needs to be fixed eventually, but I think it's fine to leave it as is in this PR.

Reply:
I added the subshell.